_src/_posts/2018-12-19-jplotlib-help-your-teams-to-leverage-data-by-using-the-perfect-graphs.markdown

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title:  "How to help your teams to leverage data by using the perfect graphs."
+date:   2018-12-19
+tag:    technology
+categories:
+    - data-science
+    - technology
+    - python
+keywords: "python, matplotlib, visualization, jplotlib"
+author: gonzalezzfelipe
+---
+
+<!--excerpt.start-->
+
+In these times, it's all about Data..
+
+... BUT, it’s not only about having your data available in a clean and easy
+data-set and feeding a machine learning algorithm, it's also about sharing it
+so other people (with more or less technical knowledge than you) can understand
+the trends without much explanation or even better, without feeling
```

Comment:
¿its implications or the trends? como para no poner que se explican solo trends?